### PR TITLE
Update ambassadors.json

### PR DESCRIPTION
**Pull request by aeberhart**
Ambassadors application for Andreas Eberhart

### CODE REVIEW

File: programs/ambassadors/ambassadors.json [hunk 1]
```diff
@@ -20,5 +20,45 @@
...
+        },
+        {
+          "type": "documentation",
+          "title": "Dashjoin Open Source & Cloud Native Low Code Development Platform",
+          "date": {
+            "year": 2024,
+            "month": "April"
+          },
+          "link": "https://dashjoin.github.io/platform/latest/"
+        }
```
Comment (benjagm) [inline programs/ambassadors/ambassadors.json:61]:
```suggestion
        {
          "type": "adopter",
          "title": "Add Dashjoin to the JSON Schema Adopters list.",
          "date": {
            "year": 2024,
            "month": "April"
          },
          "link": "https://landscape.json-schema.org/"
        },
```

I suggest we change the content of the contribution to focus on Dashjoin as adopter

**Review by benjagm — APPROVED** [review]
Thanks a lot for your application Andreas! We are so excited to have you onboard!

**Review by gregsdennis — APPROVED** [review]
Welcome Andreas!

**Review by Julian — APPROVED** [review]
Awesome!

**Review by jviotti — APPROVED** [review]

**Review by Relequestual — APPROVED** [review]
Great to have you join the Ambassadors Andreas! 🙌

**Pull request merged**